internal/dag/builder.go

```diff
@@ -518,6 +521,27 @@ func (b *Builder) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 			svhost.Secret = sec
 			svhost.MinProtoVersion = annotation.MinProtoVersion(proxy.Spec.VirtualHost.TLS.MinimumProtocolVersion)
 
+			// Check if FallbackCertificate && ClientValidation are both enabled in the same vhost
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate && tls.ClientValidation != nil {
+				sw.SetInvalid("Spec.Virtualhost.TLS.FallbackCertificate & Spec.Virtualhost.TLS.ClientValidation are incompatible together")
+				return
+			}
+
+			// If FallbackCertificate is enabled, but no cert passed, set error
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate {
+				if b.FallbackCertificate == nil {
+					sw.SetInvalid("Spec.Virtualhost.TLS.FallbackCertificate requested but the fallback Certificate Secret is not configured in Contour configuration file")
```

Comment:
```suggestion
					sw.SetInvalid("Spec.Virtualhost.TLS enabled fallback but the fallback certificate Secret is not configured in Contour configuration file")
```

for consistency with the status below

Reply:
👍